<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MailAccountDialog</class>
 <widget class="QDialog" name="MailAccountDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Accounts</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Select the directory that contains the Thunderbird profiles.</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="tbProfilesPathEdit">
        <property name="placeholderText">
         <string>Thunderbird Profiles Directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tbProfilesBrowseButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Select the mail accounts you want to monitor.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="AccountsTreeWidget" name="accountsList">
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <property name="emptyText" stdset="0">
      <string>No mail profiles were found.
Please make sure you selected the correct profiles directory.</string>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>300</number>
     </attribute>
     <column>
      <property name="text">
       <string>Email Folder</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Notification Color</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>If you monitor multiple folders, the default notification color is used to show the sum of all unread mails.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AccountsTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>mailaccountdialog.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MailAccountDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MailAccountDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
